src/ripple/app/misc/NetworkOPs.cpp

```diff
@@ -2169,88 +2156,6 @@ NetworkOPsImp::setMode(OperatingMode om)
     pubServer();
 }
 
-std::string
-NetworkOPsImp::transactionsSQL(
-    std::string selection,
-    AccountID const& account,
-    std::int32_t minLedger,
-    std::int32_t maxLedger,
-    bool descending,
-    std::uint32_t offset,
-    int limit,
-    bool binary,
-    bool count,
-    bool bUnlimited)
-{
-    std::uint32_t NONBINARY_PAGE_LENGTH = 200;
-    std::uint32_t BINARY_PAGE_LENGTH = 500;
-
-    std::uint32_t numberOfResults;
-
-    if (count)
-    {
-        numberOfResults = 1000000000;
-    }
-    else if (limit < 0)
-    {
-        numberOfResults = binary ? BINARY_PAGE_LENGTH : NONBINARY_PAGE_LENGTH;
-    }
-    else if (!bUnlimited)
-    {
-        numberOfResults = std::min(
-            binary ? BINARY_PAGE_LENGTH : NONBINARY_PAGE_LENGTH,
-            static_cast<std::uint32_t>(limit));
-    }
-    else
-    {
-        numberOfResults = limit;
-    }
-
-    std::string maxClause = "";
-    std::string minClause = "";
-
-    if (maxLedger != -1)
-    {
-        maxClause = boost::str(
-            boost::format("AND AccountTransactions.LedgerSeq <= '%u'") %
-            maxLedger);
-    }
-
-    if (minLedger != -1)
-    {
-        minClause = boost::str(
-            boost::format("AND AccountTransactions.LedgerSeq >= '%u'") %
-            minLedger);
-    }
-
-    std::string sql;
-
-    if (count)
-        sql = boost::str(
-            boost::format("SELECT %s FROM AccountTransactions "
-                          "WHERE Account = '%s' %s %s LIMIT %u, %u;") %
-            selection % app_.accountIDCache().toBase58(account) % maxClause %
-            minClause % beast::lexicalCastThrow<std::string>(offset) %
-            beast::lexicalCastThrow<std::string>(numberOfResults));
-    else
-        sql = boost::str(
-            boost::format(
-                "SELECT %s FROM "
-                "AccountTransactions INNER JOIN Transactions "
-                "ON Transactions.TransID = AccountTransactions.TransID "
-                "WHERE Account = '%s' %s %s "
-                "ORDER BY AccountTransactions.LedgerSeq %s, "
-                "AccountTransactions.TxnSeq %s, AccountTransactions.TransID %s "
-                "LIMIT %u, %u;") %
-            selection % app_.accountIDCache().toBase58(account) % maxClause %
-            minClause % (descending ? "DESC" : "ASC") %
-            (descending ? "DESC" : "ASC") % (descending ? "DESC" : "ASC") %
-            beast::lexicalCastThrow<std::string>(offset) %
-            beast::lexicalCastThrow<std::string>(numberOfResults));
-    JLOG(m_journal.trace()) << "txSQL query: " << sql;
-    return sql;
-}
-
 NetworkOPs::AccountTxs
 NetworkOPsImp::getAccountTxs(
```

Comment:
Let's remove this function from `NetworkOPs` and just call the `SQLInterface` functions directly, here and the other places where the code was moved from `NetworkOPs` to `SQLInterface`.

Reply:
Done.